mergify_engine/actions/merge_base.py

```diff
@@ -54,13 +55,38 @@ class PriorityAliases(enum.Enum):
     high = 3000
 
 
+class StrictMergeParameter(enum.Enum):
+    true = True
+    false = False
+    fasttrack = "smart+fasttrack"
+    ordered = "smart+ordered"
+
+    @classmethod
+    def __init__(cls, v):
+        if cls.__name__ not in mergify_json.JSON_TYPES:
+            mergify_json.register_type(cls)
```

Comment:
You don't need to do it at `__init__`, you can just do it when the module is loaded.